ansible/playbooks/monitoring.yml

```
@@ -201,6 +201,15 @@
                 summary: "The docs.rs deprioritized queue is unreasonably long"
                 description: "There are more than 1000 deprioritized crates in the docs.rs queue, and the situation didn't resolve itself in the 24 hours. The build queue is available at https://docs.rs/releases/queue"
 
+            - alert: HighCpuUsage
+              expr: avg by (mode) (rate(node_cpu_seconds_total{job="node",instance="docsrs.infra.rust-lang.org:9100",mode="user"}[1m])) > 0.9
```

While this query would've indeed catched the problem that happened recently, it might hide high CPU usage in some cases. The query is currently considering only the `user` mode, but a sizeable chunk of CPU usage could be due to `system` (in the kernel) or due to `iowait` (IO latency), and you'd warn to be alerted for both.

I'd suggest to instead check if there is less than 10% of idle CPU.

```suggestion
              expr: avg(rate(node_cpu_seconds_total{job="node",instance="docsrs.infra.rust-lang.org:9100",mode="idle"}[1m])) < 0.1
```